.gitignore

```diff
@@ -329,4 +329,6 @@ tools/
 
 # VisualStudioCode Patch
 # Ignore all local history of files
-.history
\ No newline at end of file
+.history
+/Dump
```

Comment:
Not sure why you need this one ?  The dump files [normally] goes below the bin folders, which is ignored.   No harm though.....

Reply:
@OsirisTerje It creates one name `D_FileThatDoesntExist.dll.dump` in the top-level folder.  Not sure which test specifically created it, but it showed up in my git changes :)